collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
First bullet - Yup. I avoided changing the versionone collector because I do not have the ability to test changes to it. However, doesn't versionone always have teams? In that case your first difference is legitimate. 

The last two bullet points happen for both versionone and jira. There is no specific logic in the UI to differentiate them. Since teams are stored in collector items there is no team to associate the component with when 'Any' is selected. The override is used as a way to get back to the collector. It is used in the feature API calls which use the feature repository which are all implementation agnostic.